File: articles/algorithms-in-public-policy-and-governance/java/ProceduralReadiness.java
public class ProceduralReadiness {
    public static void main(String[] args) {
        double dueProcess = 0.58;
        double transparency = 0.52;
        double humanReview = 0.60;
        double appealReadiness = 0.54;
        double score = (dueProcess + transparency + humanReview + appealReadiness) / 4.0;
        System.out.printf("procedural_readiness_score=%.4f%n", score);
    }
}

File: articles/algorithms-in-public-policy-and-governance/fortran/procedural_readiness.f90
program procedural_readiness
  implicit none
  real :: due_process, transparency, human_review, appeal_readiness, score
  due_process = 0.58
  transparency = 0.52
  human_review = 0.60
  appeal_readiness = 0.54
  score = (due_process + transparency + human_review + appeal_readiness) / 4.0
  print *, "procedural_readiness_score=", score
end program procedural_readiness
